templates/experimental/amazonlinux-2023.yaml

```
@@ -0,0 +1,58 @@
+minimumLimaVersion: 2.0.0
```

Probably this is going to be:

```suggestion
minimumLimaVersion: 2.1.0
```